src/FileIndexer.ts

```diff
@@ -37,6 +37,10 @@ export class FileIndexer {
     this.workingDirectoryRegExp = new RegExp(options.cwd, 'g')
   }
   public index(): void {
+    // Uncomment below if you want to skip certain files for local development.
+    // if (!this.sourceFile.fileName.includes('conflicting-const')) {
+    //   return
+    // }
```

Comment:
Why add these special comments for particular test cases? We could potentially add a test filter regex or something if that's desired, this seems a little strange.

Reply:
I haven't figured out yet how to implement this kind of filtering from the test framework so I prefer to keep it as a comment until we figure out a better solution.